Boolean.CSharp.Test/CoreTests.cs

```diff
@@ -1,24 +1,61 @@
 ﻿using Boolean.CSharp.Main;
+using Newtonsoft.Json;
 using NUnit.Framework;
 
 namespace Boolean.CSharp.Test
 {
     [TestFixture]
     public class CoreTests
     {
-        private Core _core;
+        private Customer _customer;
+        //private BankStatement _bankStatement;
 
         public CoreTests()
         {
-            _core = new Core();
-
+            _customer = new Customer();
+            //_bankStatement = new BankStatement();
         }
 
         [Test]
-        public void TestQuestion1()
+        public void addAccountTest()
         {
-
+            SavingsAccount newSavings = (SavingsAccount)_customer.createAccount(762308, accountType.savings);
+            CurrentAccount newCurrent = (CurrentAccount)_customer.createAccount(556223, accountType.current);
+            Assert.That(_customer.Accounts.Contains(newSavings));
+            Assert.That(_customer.Accounts.Contains(newCurrent));
+        }
+        [Test]
```

Comment:
can you rewrite this using `TestCase` and doing some setup in a setup method? The code is very repetitive